features/bootstrap/FeatureContext.php

```diff
@@ -468,12 +460,26 @@ public function composer_install_wp() {
 		$this->proc( 'wp core install', $install_args )->run_check();
 	}
 
-	public function create_config_composer_install( $subdir = '' ) {
-		$params = self::$db_settings;
-		$params['dbprefix'] = $subdir ? preg_replace( '#[^a-zA-Z\_0-9]#', '_', $subdir ) : 'wp_';
-		$params['skip-salts'] = true;
-		$params['extra-php'] = "require_once dirname(__DIR__) . '/vendor/autoload.php';";
-		$this->proc( 'wp core config', $params, $subdir )->run_check();
+	public function composer_add_wp_cli_local_repository() {
+		if ( !isset( $this->variables['COMPOSER_LOCAL_REPOSITORY'] ) ) {
+			$this->variables['COMPOSER_LOCAL_REPOSITORY'] = sys_get_temp_dir() . '/' . uniqid( "wp-cli-composer-local-", TRUE );
+
+			$env = self::get_process_env_variables();
+			$src = isset($env['TRAVIS_BUILD_DIR']) ? $env['TRAVIS_BUILD_DIR'] : realpath( __DIR__ . '/../../' );
```

Comment:
Yes, According to WPCS, this should be `isset( $env['TRAVIS_BUILD_DIR'] )` instead.